commands/channel_e2e_test.go

```diff
@@ -129,3 +129,95 @@ func (s *MmctlE2ETestSuite) TestUnarchiveChannelsCmdF() {
 		s.Require().Contains(printer.GetErrorLines()[0], "Unable to unarchive channel. The channel is not archived.")
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestDeleteChannelsCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	printer.SetFormat(printer.FormatPlain)
+	defer printer.SetFormat(printer.FormatJSON)
```

Comment:
This isn't strictly required.

Reply:
When I delete above code then I getting error - in lines where printer is used. Something like that:
```
    Error:          Not equal: 
                                expected: string("Deleted channel 'channel_name'")
                                actual  : *model.Channel(&model.Channel{Id:"zzgmgm8snbd8xkc3x7s1fp5mnr", CreateAt:1603220156368, UpdateAt:1603220156368, DeleteAt:0, TeamId:"", Type:"O", DisplayName:"", Name:"channel_name", Header:"", Purpose:"", LastPostAt:0, TotalMsgCount:0, ExtraUpdateAt:0, CreatorId:"a6n39sfb9tdsz8qpbcinmy4k5a", SchemeId:(*string)(nil), Props:map[string]interface {}(nil), GroupConstrained:(*bool)(nil)})

```
Theoretically, I can delete every line where printer is used - because it is unit tested, but maybe it would be better to check if the output works properly.

Reply:
`printer.GetLines` returns `printer.Lines` which is just a `[]interface{}`. With `printer.FormatJSON` the struct passed to the printer is stored in `printer.Lines` directly.

Its a bit unintuitive, but you can just compare the object directly.

So instead of `s.Require().Equal(fmt.Sprintf("Deleted channel '%s'", channel.Name), printer.GetLines()[0])`

you can write: `s.Require().Equal(channel, printer.GetLines()[0])`

This isn't a blocker, the way you've written it is fine as well.